ls1intum/Artemis
# `Development`: Solve merge conflict breaking the course statistics

### Checklist
#### General


#### Client

### Motivation, Context and Description
The combination of #4313 and #4296 left an import that breaks when it is lazily loaded and prevents the statistics page from opening.

### Steps for Testing

Prerequisites:
- 1 Student
- 1 A course the student participates in and has results in exercises

1. Log in to Artemis
2. Enter a course by clicking on the corresponding course card in Course overview
3. Try to enter the statistics tab. This should now be possible again
4. Scroll down and inspect the Performance in Exercises chart. The lines should be visible and interacting with it should also be possible again

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage
unchanged
